doc/dev/doc/network.md

```diff
@@ -13,18 +13,37 @@ You can specify multiple addresses, with or without the port information. If the
 By default, the provided list is shuffled and tried in a random order. You can disable this behaviour by setting the configuration option `hazelcast.networking.shuffleAddresses` to `false`. In this case the address list will be tried in the specified order. 
 
 ## Operation Mode
+The client has three operation modes because of the distributed nature of the data and cluster: all members, multi members and single member. All members routing is enabled by default, and is controlled by the `@Hazelcast.NetworkingOptions.RoutingMode` configuration option.
 
-The client has two operation modes because of the distributed nature of the data and cluster: smart and unisocket. Smart routing is enabled by default, and is controlled by the `hazelcast.networking.smartRouting` configuration option, but you may want to enable unisocket mode.
+> [!NOTE]
+> Smart Routing is deprecated and will be removed in the next major release. Please use the new RoutingMode mode instead.
```

Comment:
When I read that it sounded like the semantics was deprecated -- this is in reference to `hazelcast.networking.smartRouting` and not the smart routing semantics, that's just been renamed. Maybe clarify it a bit?

Reply:
I see, you are right. I've paraphrased it. How about now?